composer.json

```diff
@@ -26,12 +26,12 @@
     },
     "autoload": {
         "psr-4": {
-          "Qandidate\\Toggle\\": "lib/Qandidate/Toggle/"
+          "Qandidate\\Toggle\\": "lib/"
```

Comment:
@wjzijderveld 

What do you think about changing the directory name from `lib` to `src`?